admin_ui_support/src/Plugin/rest/resource/ConfigSchemaResource.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drupal\admin_ui_support\Plugin\rest\resource;
+
+use Drupal\Core\Config\TypedConfigManagerInterface;
+use Drupal\rest\Plugin\ResourceBase;
+use Drupal\rest\ResourceResponse;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+/**
+ * Provides a rest resource for config schema.
+ *
+ * @RestResource(
+ *   id = "config_schema",
+ *   label = @Translation("Config schema"),
+ *   uri_paths = {
+ *     "canonical" = "/admin-api/config-schema/{name}"
```

Comment:
Note: For now I went with one resource plugin without any derivative. I'm not sure there is any crucial information in config schema not safe to export.